cmd/provisioner/hostpath-provisioner.go

```diff
@@ -160,7 +160,9 @@ func (p *hostPathProvisioner) Provision(options controller.ProvisionOptions) (*v
 			},
 			Spec: v1.PersistentVolumeSpec{
 				PersistentVolumeReclaimPolicy: v1.PersistentVolumeReclaimDelete,
-				AccessModes:                   options.PVC.Spec.AccessModes,
+				AccessModes: []v1.PersistentVolumeAccessMode{
```

Comment:
Are we checking the access mode and refusing to provision if it's RWX?  Do we support ReadOnlyMany?

Reply:
So before we were just copying the access modes from the PVC, now we will make only a RWO PV. If we request a RWX then it can't match the PV with the PVC.

Reply:
Right I get it.  Nevermind about ReadOnlyMany.  I can't even see how it would be useful.